### PR TITLE
added some free next js courses

### PR DESCRIPTION
**Pull request by AmardeepKesharwani**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -661,7 +661,9 @@
 
 * [Mastering Next.js](https://masteringnextjs.com)
 * [Next.js Tutorial for Beginners](https://youtube.com/playlist?list=PLC3y8-rFHvwgC9mj0qv972IO5DmD-H0ZH) - Codevolution
-
+* [Next.js for Beginners - Full Course](https://youtu.be/1WmNXEVia8I) - Freecodecamp
+* [Routing in Next.js ](https://www.freecodecamp.org/news/how-to-setup-dynamic-routing-in-nextjs/) - Freecodecamp
```
Comment (bugbunnyy) [inline courses/free-courses-en.md:665]:
```suggestion
* [Routing in Next.js](https://www.freecodecamp.org/news/how-to-setup-dynamic-routing-in-nextjs/) - Freecodecamp
```

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -661,7 +661,9 @@
...
+* [Build Your Own Blog with Next.js and MDX](https://www.freecodecamp.org/news/how-to-build-your-own-blog-with-next-js-and-mdx/) - Freecodecamp
```
Comment (bugbunnyy) [inline courses/free-courses-en.md:666]: Provide another blank line after this course.

**Comment by bugbunnyy** [comment]
Please ensure that the courses you add to the Next.js category are alphabetically arranged.
This and the the above changes(comment) will allow you to solve linter warnings.

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -661,6 +661,9 @@
 
 * [Mastering Next.js](https://masteringnextjs.com)
 * [Next.js Tutorial for Beginners](https://youtube.com/playlist?list=PLC3y8-rFHvwgC9mj0qv972IO5DmD-H0ZH) - Codevolution
+* [Next.js for Beginners - Full Course](https://youtu.be/1WmNXEVia8I) - Freecodecamp
+* [Routing in Next.js](https://www.freecodecamp.org/news/how-to-setup-dynamic-routing-in-nextjs/) - Freecodecamp
```
Comment (eshellman) [inline courses/free-courses-en.md:665]:
```suggestion
```
It's an article, not a course

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -661,6 +661,9 @@
...
+* [Build Your Own Blog with Next.js and MDX](https://www.freecodecamp.org/news/how-to-build-your-own-blog-with-next-js-and-mdx/) - Freecodecamp
```
Comment (eshellman) [inline courses/free-courses-en.md:666]:
```suggestion
```
it's an article, not a course

**Pull request merged**